src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function defaultCompare(a, b) {
+    if (a.toString() < b.toString()) {
+      return -1;
+    }
+
+    if (a.toString() > b.toString()) {
+      return 1;
+    }
```

Comment:
Good job. But I think you should use a variable for a.toString() and b.toString() to avoid code duplication.